### PR TITLE
Enable clients without telemetry enabled

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

**Comment by robhudson** [comment]
Is there a bug number or issue to reference on this change?

File: tests/dataset.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def generate_payload(dimensions, aggregated_child_histograms):
     environment = {
         u"system": {u"os": {u"name": dimensions["os"],
                             u"version": dimensions["os_version"]}},
-        u"settings": {u"telemetryEnabled": True,
+        u"settings": {u"telemetryEnabled": False,
```
Comment (robhudson) [inline tests/dataset.py]: Do we still need this in the test data if it's not being checked?

**Comment by fbertsch** [comment]
I suppose the relevant bug is 1477348

File: tests/dataset.py (hunk 1 above)
Comment (fbertsch) [reply tests/dataset.py]: This now verifies that the non-telemetry enabled clients are aggregated correctly, so it's probably good to keep in

**Pull request merged**